tests/unit/test_progress.py

```diff
@@ -10,6 +11,14 @@ def test_quiet(caplog, capsys):
         assert out_err.out == ""
         assert out_err.err == ""
     with caplog.at_level(logging.INFO, logger="dvc"):
+        for _ in Tqdm(range(10)):
+            pass
+        out_err = capsys.readouterr()
+        assert out_err.out == ""
+        assert out_err.err == ""
+
+        # simulate interactive terminal
+        sys.stderr.isatty = lambda: True
```

Comment:
We should fix this afterwards or use mock.

Reply:
yes not somethign for this PR to handle

Reply:
@casperdcl But why are we modifying `sys.stderr` directly? We not mock here?

Reply:
94a3fa31